src/consensus/params.h

```diff
@@ -88,6 +90,8 @@ struct Params {
     int TaprootHeight;
     /** Don't warn about unknown BIP 9 activations below this height.
      * This prevents us from warning about the CSV and segwit activations. */
+    /** Block height at which annex (BIPXXX) becomes active */
+    int AnnexHeight = std::numeric_limits<int>::max();
```

Comment:
`FooHeight` is for deployments that are active on mainnet; they're not needed for `DeploymentPos` (the height is determined dynamically by signalling)

Reply:
Corrected.

Reply:
Still plenty of `AnnexHeight` references; maybe run "make" before "git push" ? :wink:

Reply:
Yeah lol, run make this time and grep for `AnnexHeight` references.